javascript/javascript1/week3/hyf-homework-week3/remove.js

```diff
@@ -0,0 +1,18 @@
+
+
+const names = ["Peter", "Ahmad", "Yana", "kristina", "Rasmus", "Samuel", "katrine","Tala"];
+const nameToRemove = "Ahmad";
```

Comment:
This code just work for specific name that you desidec to remove but it must be able to find whatever name that we chose to remove. Please write that dynamic and let me know if you have any question about how to do it.

Reply:
I have done changes, can you confirm me to merge?